src/diffusers/models/attention_processor.py

```diff
@@ -741,11 +738,14 @@ def __call__(
         encoder_hidden_states: Optional[torch.FloatTensor] = None,
         attention_mask: Optional[torch.FloatTensor] = None,
         temb: Optional[torch.FloatTensor] = None,
-        scale: float = 1.0,
+        *args,
+        **kwargs,
     ) -> torch.Tensor:
-        residual = hidden_states
+        if len(args) > 0 or kwargs.get("scale", None) is not None:
+            deprecation_message = "The `scale` argument is deprecated and will be ignored. Please remove it, as passing it will raise an error in the future. `scale` should directly be passed while calling the underlying pipeline component i.e., via `cross_attention_kwargs`."
```

Comment:
what does this mean? aren't they ignored when passed via `cross_attention_kwargs`?
> `scale` should directly be passed while calling the underlying pipeline component i.e., via `cross_attention_kwargs`."

Reply:
We have test here for that:

https://github.com/huggingface/diffusers/blob/a1cb106459da4c595d22c04e026d7169d8dcfd2b/tests/lora/test_lora_layers_peft.py#L234

Perhaps, @younesbelkada could clarify this a bit?

Reply:
Ah I see. It's only for the text encoder. Younes, would you explain how `scale` is then handled for UNet?

Reply:
Oh there's actually `test_simple_inference_with_text_unet_lora_and_scale()`: 

https://github.com/huggingface/diffusers/blob/a1cb106459da4c595d22c04e026d7169d8dcfd2b/tests/lora/test_lora_layers_peft.py#L555

This should confirm UNet scale is also effective.

Reply:
Ran `test_integration_logits_with_scale()` in our SLOW test environment as well and it passes!

```bash
RUN_SLOW=1 pytest tests/lora/test_lora_layers_peft.py -k "test_integration_logits_with_scale"
```

Reply:
yes ! indeed `scale` should be properly handled under the hood by peft. Scale corresponds to `lora_alpha / lora_r` in the peft config

Reply:
ohh thanks!
i see, it's here https://github.com/huggingface/diffusers/blob/2ea28d69dc3ac72d97e31f51e81d37a5045acba5/src/diffusers/models/unets/unet_2d_condition.py#L1191